static/js/topic_list_data.js

```diff
@@ -31,6 +34,12 @@ export function get_list_info(stream_id, zoomed) {
         const num_unread = unread.num_unread_for_topic(stream_id, topic_name);
         const is_active_topic = active_topic === topic_name.toLowerCase();
         const is_topic_muted = muted_topics.is_topic_muted(stream_id, topic_name);
+        const resolved = topic_name.startsWith(message_edit.RESOLVED_TOPIC_PREFIX);
+        let topic_display_name = topic_name;
+
+        if (resolved) {
+            topic_display_name = _.trimStart(topic_name, message_edit.RESOLVED_TOPIC_PREFIX);
```

Comment:
Hmm, I think we don't want to use `trimStart` here -- we should specifically just remove a leading `RESOLVED_TOPIC_PREFIX`.  (The situations where you have some other weird sequence of checks and spaces are unlikely, but in those cases we want to still display what the user provided).